.yamato/project-publish.yml

```diff
@@ -17,12 +17,24 @@ publish_{{ project.name }}_{{ package.name }}:
         - "upm-ci~/packages/*.tgz"
   dependencies:
     - .yamato/project-pack.yml#pack_{{ project.name }}
-    - .yamato/project-tests.yml#validate_{{ project.name }}_{{ package.name }}_{{ test_platforms.first.name }}_{{ validation_editor }}
```

Comment:
I was going to call out that we probably didn't want to remove the validation dependency from this job... but it looks like whatever job this was pointing to doesn't actually exist anymore...

Can you fix this up to depend on the same validation job you have for the dry-run?

Reply:
The validation job is still there look further down in th diff. I removed the requirements of all the test jobs and fixed the validation job name being wrong.

Reply:
I see it down on line 37 in the dry-run job, but not in this, the non-dry-run. Won't publish fail if we haven't run validation yet? Seems odd for us to require a 2-step both-manual process to publish.